lib/internal/loader/ModuleRequest.js

```diff
@@ -90,11 +98,10 @@ loaders.set('json', async (url) => {
 });
 
 exports.resolve = (specifier, parentURL) => {
-  if (NativeModule.nonInternalExists(specifier)) {
-    return {
-      url: specifier,
-      format: 'builtin'
-    };
+  if (/^node:/.test(parentURL) && parentURL.slice(5) === specifier) {
+    return { url: `${specifier}.js`, format: 'builtin/cjs' };
```

Comment:
When does this case ever happen?

Reply:
when the wrapper imports the actual cjs it looks like `node:assert` calling `assert`, and thats the only time the builtin cjs should get used